docs/src/userguide/loading_iris_cubes.rst

```diff
@@ -215,6 +215,14 @@ constraint to ``load``::
     level_10 = iris.Constraint(model_level_number=10)
     cubes = iris.load(filename, forecast_6 & level_10)
 
+.. note::
+
+    Whilst ``&`` is supported, the ``|`` that might reasonably be expected
+    is not. This is because "or" constraints could lead to a cube which did
```

Comment:
It might make sense to go into the details of why "|"/"or" is not supported in the docstring of the Constraint class and refer to that documentation here, thus saving space in the user guide and giving you freedom to expand the explanation of Constaint combination.

Reply:
Have done this